curation_concerns-models/app/models/concerns/curation_concerns/upload_set_behavior.rb

```diff
@@ -7,7 +7,6 @@ module UploadSetBehavior
     included do
       has_many :file_sets, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.isPartOf
 
-      property :creator, predicate: ::RDF::DC.creator
```

Comment:
There are associations between them and the files that are uploaded.  The files that are uploaded have the owners.  Can we infer their ownership through that?  UploadSets have no metadata, and are not editable, or deletable, they're basically just empty resources, used for a URL, right?  Why do we need to make them owned by anyone?